pkg/apis/pipeline/v1/taskrun_types.go

```diff
@@ -71,14 +71,14 @@ type TaskRunSpec struct {
 	// This field is only supported when the alpha feature gate is enabled.
 	// +optional
 	// +listType=atomic
-	StepOverrides []TaskRunStepOverride `json:"stepOverrides,omitempty"`
```

Comment:
@lbernick could you help me understand why we only change the names in `taskrun.spec` instead also for [PipelineTaskRunSpec](https://github.com/tektoncd/pipeline/blob/d15e5c4b0430ccef5db667f88338cecd8ef4a420/pkg/apis/pipeline/v1/pipelinerun_types.go#L526)? (also why I hesitated to update this PR)

Reply:
because I made a mistake in the TEP :) you can go ahead and rename all of them